master/reference/calicoctl/resources/bgpconfig.md

```diff
@@ -0,0 +1,47 @@
+---
+title: BGP Configuration Resource (BGPConfiguration)
+---
+
+A BGP configuration resource (BGPConfiguration) represents BGP specific configuration options for the cluster.
+
+For `calicoctl` commands that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `bgpconfiguration`, `bgpconfig`, `bgpconfigurations`, `bgpconfigs`.
+
+### Sample YAML
+
+```yaml
+apiVersion: projectcalico.org/v2
+kind: BGPConfiguration
+metadata:
+  name: default
+spec:
+  logSeverityScreen: info
+  nodeToNodeMeshEnabled: true
+  asNumber: 63400
+```
+
+### BGP Peer Definition
+
+#### Metadata
+
+| Field       | Description                 | Accepted Values   | Schema |
+|-------------|-----------------------------|-------------------|--------|
+| name     | Unique name to describe this resource instance. Must be specified. | Alphanumeric string with optional `.`, `_`, `-`, or `/` | string |
+
+- Resource with the name "default" has a specific meaning - this contains the BGP global default configuration overrides.
```

Comment:
I put a comment here earlier, but one of your commits erased it, so will put it again, but different this time, because now I notice that the name must be unique. So then, I would change this sentence to be "If named `default`, the resource overrides the BGP global default configuration settings." 

I must say, I am not sure, but to the degree that we explain somewhere in the docs what the "BGP global default configuration settings" are then we should really make that into a hyperlink so that they can click it to discover what these are.

Reply:
:) github folds the comment unfortunately when the code is modified, but I can still access the comment. That sentence is not quite right, global config doesn't override anything, it's just the default config. I had an extra override at the end which was wrong.

As far as repeating this, this is only applicable to 2 resources so it might be ok, especially since we might get rid of node-specific config in future